images-list

```diff
@@ -34,3 +34,8 @@ us.gcr.io/k8s-artifacts-prod/external-dns/external-dns rancher/kubernetes-extern
 openpolicyagent/gatekeeper rancher/opa-gatekeeper v3.1.0-beta.11
 k8s.gcr.io/k8s-dns-node-cache rancher/k8s-dns-node-cache 1.15.7
 sonobuoy/sonobuoy rancher/sonobuoy-sonobuoy v0.16.3
+banzaicloud/logging-operator rancher/logging-operator 3.4.0
+jimmidyson/configmap-reload rancher/configmap-reload v0.2.2
+paynejacob/fluent-bit-out-syslog rancher/fluent-bit-out-syslog latest
```

Comment:
If this image doesn't actually exist anywhere, then lets put version at 0.1.0

Reply:
Changed to 0.1.0, this image can be dropped once we have our own ci but until that is ready we can mirror my build.